### PR TITLE
Rename `ComponentGraph.components()` parameters to be plural

### PR DESCRIPTION
**Pull request by Marenz**

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -20,6 +20,9 @@
 
 - The `Fuse` class has been moved to the `frequenz.sdk.timeseries` module.
 
+- The `ComponentGraph.components()` parameters `component_id` and `category_id`
+  were renamed to `component_ids` and `category_ids`, respectively.
```
Comment (llucax) [inline RELEASE_NOTES.md]: Wrong name `category_id`(s) and also GitHub release notes doesn't like line breaks.

```suggestion
- The `ComponentGraph.components()` parameters `component_id` and `component_category`   were renamed to `component_ids` and `component_categories`, respectively.
```

**Comment by Marenz** [comment]
fixed

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -20,6 +20,9 @@
 
 - The `Fuse` class has been moved to the `frequenz.sdk.timeseries` module.
 
+- The `ComponentGraph.components()` parameters `component_id` and `category_id`
+  were renamed to `component_category` and `component_categories`, respectively.
```
Comment (llucax) [inline RELEASE_NOTES.md]: You still kept the `\n` :cry:

**Review by llucax — COMMENTED** [review]
I would be fine with merging as is and fix later in the GitHub release, but I would still like to know if at least @matthias-wende-frequenz or @shsms are OK with it, so delaying the approval.

File: RELEASE_NOTES.md (hunk 2 above)
Comment (llucax) [reply RELEASE_NOTES.md]: Well, actually you also forgot to change one `category_id` :scream:

Comment (llucax) [reply RELEASE_NOTES.md]: Actually you messed up completely... :rofl:

Comment (Marenz) [reply RELEASE_NOTES.md]: yes 😆  but to be fair your original change request also suggested something wrong :P

**Review by shsms — DISMISSED** [review]

**Comment by Marenz** [comment]
Forgot to remove the \n. Now it's in one line.

**Review by shsms — APPROVED** [review]

**Pull request merged**